src/Lykke.Service.Decred.Api/Controllers/TransactionController.cs

```diff
@@ -0,0 +1,191 @@
+                return Json(new
+                {
+                    errorCode = "notEnoughBalance",
+                    transactionContext = (string) null
+                });
+            }
+
+            catch (BusinessException ex) when (ex.Reason == ErrorReason.DuplicateRecord)
+            {
+                // If this operationid was already broadcast, return 409 conflict.
+                return await GenericErrorResponse(ex, request.OperationId, HttpStatusCode.Conflict);
+            }
+        }
+
+        [HttpPost("api/transactions/broadcast")]
+        public async Task<IActionResult> Broadcast([FromBody] BroadcastTransactionRequest request)
+        {
+            try
+            {
+                // Broadcast the signed transaction
+                await _txBroadcastService.Broadcast(request.OperationId, request.SignedTransaction);
+                Response.StatusCode = (int) HttpStatusCode.OK;
+                return Json(new { errorMessage = "" });
+            }
+            catch (BusinessException ex) when (ex.Reason == ErrorReason.DuplicateRecord)
+            {
+                // If this operationid was already broadcast, return 409 conflict.
+                return await GenericErrorResponse(ex, request.OperationId, HttpStatusCode.Conflict);
+            }
+        }
+
+        [HttpGet("api/transactions/broadcast/single/{operationId}")]
+        public async Task<IActionResult> GetBroadcastedSingleTx(Guid operationId)
+        {
+            if(operationId == Guid.Empty)
+                throw new BusinessException(ErrorReason.BadRequest, "Invalid operation id");
+
+            try
+            {
+                // Retrieves a broadcasted transaction
+                var result = await _txBroadcastService.GetBroadcastedTxSingle(operationId);
+                return Json(result);
+            }
+            catch (BusinessException e) when (e.Reason == ErrorReason.RecordNotFound)
+            {
+                return NoContent();
+            }
+
+            catch (Exception e)
+            {
+                return await GenericErrorResponse(e, operationId, HttpStatusCode.BadRequest);
+            }
+        }
+
+        [HttpDelete("api/transactions/broadcast/{operationId}")]
+        public async Task<IActionResult> RemoveObservableOperation(Guid operationId)
+        {
+            if (operationId == Guid.Empty)
+                throw new BusinessException(ErrorReason.BadRequest, "Operation id is invalid");
+
+            try
+            {
+                await _txBroadcastService.UnsubscribeBroadcastedTx(operationId);
+                return Ok();
+            }
+            catch (BusinessException e) when(e.Reason == ErrorReason.RecordNotFound)
+            {
+                return NoContent();
+            }
+            catch (Exception e)
+            {
+                return await GenericErrorResponse(e, operationId, HttpStatusCode.InternalServerError);
```

Comment:
Please, use [this](https://github.com/LykkeCity/Lykke.Service.BlockchainApi.Client/blob/master/src/Lykke.Service.BlockchainApi.Contract/BlockchainErrorResponse.cs) model for error responses (here and in other places)